pkg/api/steve/additionalapi.go

```diff
@@ -19,11 +19,14 @@ import (
 )
 
 func AdditionalAPIsPreMCM(config *wrangler.Context) func(http.Handler) http.Handler {
+	connectHandler := configserver.New(config)
 	if features.RKE2.Enabled() {
 		mux := gmux.NewRouter()
 		mux.UseEncodedPath()
-		mux.Handle("/v3/connect/agent", configserver.New(config))
+		mux.Handle("/v3/connect/agent", connectHandler)
+		mux.Handle("/v3/connect/config-yaml", connectHandler)
 		mux.Handle("/system-agent-install.sh", server.InstallHandler())
+		mux.Handle("/windows-agent-install.ps1", server.WindowsInstallHandler())
```

Comment:
Can this be combined like the `connectHandler` above?

Reply:
two different ideas too me... ones a config server returning json and needs controllers/caches/settings etc. and the other returns plain text scripts to be consumed by down stream nodes